### PR TITLE
Minor changes for updated VAT reference testing [VS-1054]

### PR DESCRIPTION
**Pull request by mcovarr**
- Successful [VAT creation](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/9503ad5e-676f-4e48-90ac-44022bc4d608)
- Reasonably successful [VAT validation](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/9b4dc893-cad5-4204-b40d-3d3f0a541db3)? Could use a check on this

### CODE REVIEW

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 1]
```diff
@@ -52,11 +53,13 @@ workflow GvsQuickstartIntegration {
...
-            full_interval_list = full_wgs_interval_list,
-            chromosomes = ["chrX", "chrY", "chr20"],
-            variants_docker = effective_variants_docker,
+    if (chr20_X_Y_only) {
+        call FilterIntervalListChromosomes {
+            input:
+                full_interval_list = full_wgs_interval_list,
+                chromosomes = ["chrX", "chrY", "chr20"],
+                variants_docker = effective_variants_docker,
+        }
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:62]: what was the reasoning behind adding this as an option?

Comment (mcovarr) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:62]: I ran into a failure that I thought was due to Nirvana not being happy about all the missing chromosomes. Unfortunately I can't seem to find the error I ran into in job history so I'm going to re-make the Quickstart VDS with only 20, X and Y to make sure that was really the problem.

Comment (mcovarr) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:62]: OK here was the problem: we split up our processing by chromosomes, and Nirvana was [mightily displeased](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/69258dbd-2341-4e11-a128-b85c766af792) to find no genes associated with the missing chromosomes (see most `AnnotateVCF` shards):

```
ERROR: The provided TSV has no valid custom annotation entries.
```

File: scripts/variantstore/variant_annotations_table/README.md [hunk 2]
```diff
@@ -10,9 +10,11 @@ The pipeline takes in a Hail Variant Dataset (VDS), creates a queryable table in
...
 
-You will want to set up a [notebook in a similar configuration](../docs/aou/vds/cluster/AoU%20Delta%20VDS%20Cluster%20Configuration.md) as you did for creating the VAT (minus the custom wheel) and then copy two python scripts for running locally in the notebook. The two scripts are:
+Set up a notebook in a configuration suitable for working with a VDS of the size being used as the input to VAT creation. The steps followed during the
+Delta callset for [creating a Hail cluster in a notebook](../docs/aou/vds/cluster/AoU%20Delta%20VDS%20Cluster%20Configuration.md) should be helpful here, adjusting the number of workers and preemptibles as needed.
+Once the cluster is up and running, copy two python scripts for running locally in the notebook:
 
-* `create_vat_inputs.py` which you can get via `curl -O https://raw.githubusercontent.com/broadinstitute/gatk/ah_var_store/scripts/variantstore/wdl/extract/create_vat_inputs.py`
+* `create_vat_inputs.py` which you can get via the following command, replacing `your_branch_name` as appropriate: `curl -O https://raw.githubusercontent.com/broadinstitute/gatk/your_branch_name/scripts/variantstore/wdl/extract/create_vat_inputs.py`
```
Comment (RoriCremer) [inline scripts/variantstore/variant_annotations_table/README.md:17]: should we put this in our docker instead?

Comment (RoriCremer) [reply scripts/variantstore/variant_annotations_table/README.md:17]: oh wait, this still isn't in a wdl, I'm getting ahead of myself!

Comment (mcovarr) [reply scripts/variantstore/variant_annotations_table/README.md:17]: Not here as we're not using a Docker, but perhaps @rsasch might do that in her "WDLize the sites only-making" work?

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:62]: ah, ok — thanks for finding that out!

**Review by rsasch — APPROVED** [review]

**Pull request merged**